openapi-bal-service/src/main/java/io/ballerina/openapi/converter/service/OpenAPIComponentMapper.java

```diff
@@ -178,20 +178,13 @@ private ObjectSchema generateObjectSchemaFromRecordFields(Map<String, Schema> sc
             }
             String type = field.getValue().typeDescriptor().typeKind().toString().toLowerCase(Locale.ENGLISH);
```

Comment:
unused variable `type`

Reply:
This `type` used for getting schema in `getOpenApiSchema()` method.